web-registry/src/components/templates/PortfolioTemplate.tsx

```diff
@@ -28,13 +33,18 @@ const useStyles = makeStyles((theme: Theme) => ({
       marginBottom: theme.spacing(4.25),
     },
   },
+  arrow: {
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
 }));
 
 export const PortfolioTemplate: React.FC<PortfolioTemplateProps> = ({
-  accountAddress,
+  credits,
+  basketTokens,
+  children,
   renderCreditActionButtons,
   renderBasketActionButtons,
-  children,
 }) => {
   const styles = useStyles();
 
```

Comment:
I like how you updated this to be more of a presentation component - logic handled elsewhere.